src/components/search-input/search-input.module.scss

```diff
@@ -0,0 +1,46 @@
+.form {
+    position: relative;
+    left: 1em;
+    width: 16em;
+
+    @media (max-width: 768px) {
+        width: 10em;
+    }
```

Comment:
This makes it not a general-purpose search input, but a search input for the specific header design.

So let's move this part along with `left: 1em; and width: 16em;` into the header CSS (my preferred option), or call this component HeaderSearchInput to make it more clear that you're not supposed to use it for anything else.

Reply:
moved